README.md

```diff
@@ -135,16 +135,19 @@ registry with data is to publish
 [deno_std](https://github.com/denoland/deno_std) to the registry. This can be
 done via the following steps:
 
-1. Clone https://github.com/denoland/deno_std in the same parent folder as the
-   `jsr` project
-2. Make sure to [make yourself a staff user/admin](#making-yourself-a-staff-useradmin)
-3. Run `JSR_URL=http://jsr.test deno publish` to publish all of the @std
+1. Make sure to [make yourself a staff user/admin](#making-yourself-a-staff-useradmin).
+2. Run `SELECT id, name FROM users;` in the `registry` database to get your user's UUID.
```

Comment:
I'd rather point to goign to your profile page and copying the UUID from the URL, as its much simpler than connecting to the DB

Reply:
true, valid point.
was referring to the db as we'd list the steps on how to do so in the "making yourself a staff user" anyway.

updated as suggested.